src/main/scala-2.12/com/linagora/gatling/imap/action/ActionActor.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2011-2019 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
Please add the URL of where this was copied from (and why)

Reply:
Pretty much it's classes we were using but that disappeared on version upgrades of Gatling. Was easier for the migration to backport them :)

I will add a little comment :)